.github/workflows/stale.yml

```
@@ -0,0 +1,27 @@
+name: Stale issues
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+permissions:
+  actions: write
+  contents: write # only for delete-branch option
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close Stale Issues
+        uses: actions/stale@v9.1.0
+        with:
+          days-before-stale: 90
+          days-before-close: 14
+          stale-issue-label: "stale"
+          stale-pr-label: "stale"
+          close-issue-label: "closed"
+          close-pr-label: "closed"
+          days-before-pr-close: 30
```

Prefer reducing this to just issues for now.
```suggestion
          close-issue-label: "closed"
```

---

Will update - proposed change ends up w action just using default PR close delays